dot/rpc/modules/dev_test.go

```diff
@@ -2,3 +2,306 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 package modules
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	apimocks "github.com/ChainSafe/gossamer/dot/rpc/modules/mocks"
```

Comment:
nit why not just call them mocks and remove the `apimocks` alias? :thinking:

Reply:
resolved!